stable/redmine/values.yaml

```
@@ -302,6 +302,16 @@ resources: {}
   #   cpu: "2"
   #   memory: "1G"
 
+## Pods Service Account
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+serviceAccount:
+  ## Specifies whether a ServiceAccount should be created
+  ##
+  create: false
```

```suggestion
  create: true
```
https://helm.sh/docs/topics/chart_best_practices/rbac/